F77.pm

```diff
@@ -53,8 +53,6 @@ warn "\nExtUtils::F77: Version $VERSION\n";
 # hash keys. First key is usually the name of the architecture as
 # returned by Config (modulo ucfirst()).
 
-print "Loaded ExtUtils::F77 version $VERSION\n";
```

Comment:
I would prefer to leave this in. Since EU::F77 is an install-time utility I always like to know what version is being used, without having to set a debug flag,

Reply:
Updated to restore this.